src/trackers/classes/site.js

```diff
@@ -31,6 +35,22 @@ class Site {
         this.cnameCloaks = {}
 
         this.analyzeRequest = _analyzeRequest.bind(this)
+
+        this.documentCookies = Object.values(siteData.data.apis.savedCalls
+            .filter(call => {
+                if (!isSavedCookieSetterCall(call) || !call.source.startsWith('http')) {
```

Comment:
`!call.source.startsWith('http')` - hm, when does this happen?  Are there any legitimate cases or just misattributions in the crawl data?

Reply:
I some cases when `call.source == 'about:blank'` in the data.

Reply:
Ah, gotcha. AFAIK this means that the main document was the source (inline script in the main doc), but I'd need to double check.